neost/__init__.py

```diff
@@ -1,6 +1,6 @@
 """ NEoST: Open-source code for equation of state inference via nested sampling. """
 from __future__ import print_function
-__version__ = "0.9.0"
+__version__ = "0.10.0"
 __author__ = "Geert Raaijmakers"
```

Comment:
I think you should change author to "The NEoST core team'.  This is consistent with what we did for X-PSI and also far more accurate!!